GL/index.js

```diff
@@ -46,7 +46,7 @@ app.delete('/deletePerson', function(req, res){
     
     axios.delete(url)
         .then((response) => {
-            console.log(`${response.data}`)
+            console.log(`response.data`)
```

Comment:
on line 49, remove the backticks ` and it will show the intended info.

Reply:
Agree with Jonathan, right now it displays "response.data" literally instead of the actual data.